src/main/webapp/scripts/group-script.js

```diff
@@ -22,15 +28,147 @@ function getAllGroups(){
 
 /* Fill in informational fields on the sidebar of the page */
 function loadSidebar(groups) {
-    
+    const domainName = document.getElementById("domain-name");
+    domainName.innerHTML = "@" + domain;
+
+    const numGroups = document.getElementById("num-groups");
+    numGroups.innerHTML = groups.length;
+}
+
+function visualize() {
+    var color = d3.scaleLinear()
+    .domain([0, 5])
+    .range(["hsl(152,80%,80%)", "hsl(228,30%,40%)"])
+    .interpolate(d3.interpolateHcl)
+
+    var format = d3.format(",d")
+
+    var width = 932
```

Comment:
you might want to size this based on the size of the browsers window something like https://www.w3schools.com/jsref/prop_win_innerheight.asp, so that larger screens can take advantage of more screen space

Reply:
I guess a problem with the d3 packing a set height and width is that if the user resizes their window then the graph won't change, but I have changed it to use innerwidth for now.